src/CernerOneIcons-CSV.csv

```diff
@@ -1,255 +1,255 @@
-bedRequested,src/icons/0169_bed_a.svg,themeable ,
-bedAssigned,src/icons/0170_checkBed_i_heavyCircleGreen.svg,,
-bedAssigned_low_light,src/icons/LL_0170_checkBed_i_heavyCircleGreen.svg,,
-dischargeComplete,src/icons/0171_checkdoorOpen_i_heavyCircleGreen.svg,,
-dischargeComplete_low_light,src/icons/LL_0171_checkdoorOpen_i_heavyCircleGreen.svg,,
-dischargeComingDue,src/icons/0172_exclamationDoorOpen_i_heavyTriangleYellow.svg,,
-dischargeComingDue_low_light,src/icons/LL_0172_exclamationDoorOpen_i_heavyTriangleYellow.svg,,
-dischargeOverDue,src/icons/0173_exclamationDoorOpen_i_heavyTriangleRed.svg,,
-dischargeOverDue_low_light,src/icons/LL_0173_exclamationDoorOpen_i_heavyTriangleRed.svg,,
-flipVertical,src/icons/0174_twoTrianglesRight_a.svg,themeable ,
-flipHorizontal,src/icons/0174_twoTrianglesUp_a.svg,themeable ,
-rotateLeft,src/icons/0175_squareCurvedArrowLeft_a.svg,themeable ,
-rotateRight,src/icons/0175_squareCurvedArrowRight_a.svg,themeable ,
-hold,src/icons/0176_stopwatch_pause_i.svg,themeable ,
-fitWidth,src/icons/0177_arrowsLeftRightLines_a.svg,themeable ,
-fitScale,src/icons/0178_arrowsUpDownRightLeftLines_a.svg,themeable ,
-drugDrug,src/icons/0179_pill_exclamation_i_heavyTriangleYellow.svg,,
-drugDrug_low_light,src/icons/LL_0179_pill_exclamation_i_heavyTriangleYellow.svg,,
-secondaryProvider,src/icons/0180_people_a_light.svg,themeable ,
-partiallyMet,src/icons/0181_halfCircle_i.svg,themeable ,bi-directional
-simulation,src/icons/0182_monitor_a.svg,themeable ,
-collapseRow,src/icons/0183_arrowsInTwoLines_a.svg,themeable ,
-expandRow,src/icons/0184_arrowsOutTwoLines_a.svg,themeable ,
-doubleChevronLeft,src/icons/0185_twoChevronsLeft_a.svg,themeable ,
-doubleChevronRight,src/icons/0185_twoChevronsRight_a.svg,themeable ,
-circle_symbol,src/icons/S_0001_circle.svg,themeable ,
-diamond_symbol,src/icons/S_0002_diamond.svg,themeable ,
-plus_symbol,src/icons/S_0003_plus.svg,themeable ,
-rectangle_symbol,src/icons/S_0004_rectangle.svg,themeable ,
-square_symbol,src/icons/S_0005_square.svg,themeable ,
-teardrop_symbol,src/icons/S_0006_teardrop.svg,themeable ,
-teardropUp_symbol,src/icons/S_0007_teardropUp.svg,themeable ,
-thinDiamond_symbol,src/icons/S_0008_thinDiamond.svg,themeable ,
-triangle_symbol,src/icons/S_0009_triangle.svg,themeable ,
-triangleDown_symbol,src/icons/S_0010_triangleDown.svg,themeable ,
-x_symbol,src/icons/S_0011_x.svg,themeable ,
-circleLines_symbol,src/icons/S_0012_circleLines.svg,themeable ,
-catalogueNumber,src/icons/R_catalogueNumber.svg,themeable ,
-caution,src/icons/R_caution.svg,themeable ,
-consultInstructionsForUse,src/icons/R_consultInstructionsForUse.svg,themeable ,
-dateOfManufacture,src/icons/R_dateOfManufacture.svg,themeable ,
-manufacturer,src/icons/R_manufacturer.svg,themeable ,
\ No newline at end of file
+add,src/icons/0001_plus_a.svg,themeable ,,,Deprecating,
+search,src/icons/0002_magnifyingGlass_a.svg,themeable ,,,reload,src/icons/0152_circleArrow_a.svg

```

Comment:
Theres some odd formatting with this, I think you'll want to update the CSV to this: https://gist.githubusercontent.com/bjankord/67ecee693fae7ae29d6c5c650bbaecaa/raw/7e34befbec6421c67402a7e8230d5a342b6c225d/icons.csv

Reply:
Updated CSV for formatting